war/src/main/less/base/style.less

```diff
@@ -1255,15 +1245,22 @@ div.to-be-removed { display: none; }
 
 textarea {
     margin-bottom: 0;
+    resize: vertical;
 }
 
+// Remove styling + textarea-handle.gif when IE11 is no longer supported
 div.textarea-handle {
-    height: 5px;
+    height: 6px;
     font-size: 0;
     background: var(--light-grey) url(../../images/textarea-handle.gif) no-repeat 50% 1px;
     border: 1px solid var(--medium-grey);
     border-top: none;
     cursor: s-resize;
+
+    // Hide .textarea-handle on devices that support native textarea resizing
+    @supports (resize: both) {
+        display: none;
```

Comment:
It seems that by removing this you have to scroll all the way to the right to resize now, which is not so obvious, possibly because the text area is so large.

It also seems that you can no longer horizontally resize which is probably fine I don't think we need that.

any thoughts?

Reply:
I think this will be fixed by making the form less wide